fog/ingest/server/src/controller_state.rs

```diff
@@ -268,6 +273,20 @@ impl IngestControllerState {
             }
         }
     }
+
+    pub fn get_last_report_published_block_index(&self) -> u64 {
+        self.last_report_published_block_index
+    }
+
+    /// Set the last block that an ingest report was published for
+    pub fn set_last_report_published_block_index(&mut self, new_val: u64) {
+        self.last_report_published_block_index = new_val;
+    }
+
+    /// Check is the given block index the last block that an ingest report was published for
+    pub fn is_last_report_published_block_index(&self, block_index: u64) -> bool {
```

Comment:
Maybe a name like `has_report_expired` would be slightly more clear

Reply:
uneeded since using get-status now